Data files are installed under %(installdir)s/share and not under site-packages
where pkg_resources expects them, this fixes that in the simplest way possible.

Author: Jakub Zárybnický

diff -ru dv3d-8.2.1/DV3D/ButtonBarWidget.py dv3d-8.2.1-edit/DV3D/ButtonBarWidget.py
--- dv3d-8.2.1/DV3D/ButtonBarWidget.py	2020-07-17 23:18:27.000000000 +0200
+++ dv3d-8.2.1-edit/DV3D/ButtonBarWidget.py	2021-03-02 17:14:57.952866577 +0100
@@ -9,7 +9,7 @@
 import numpy as np
 from .ConfigurationFunctions import *
 import pkg_resources
-dv3d_egg_path = pkg_resources.resource_filename(pkg_resources.Requirement.parse("dv3d"), "share/dv3d")
+dv3d_egg_path = pkg_resources.resource_filename(pkg_resources.Requirement.parse("dv3d"), "../../../share/dv3d")
 ButtonDir = os.path.join( dv3d_egg_path, 'buttons' )
 
 class OriginPosition:
diff -ru dv3d-8.2.1/DV3D/ColorMapManager.py dv3d-8.2.1-edit/DV3D/ColorMapManager.py
--- dv3d-8.2.1/DV3D/ColorMapManager.py	2020-07-17 23:18:27.000000000 +0200
+++ dv3d-8.2.1-edit/DV3D/ColorMapManager.py	2021-03-02 17:15:05.905834592 +0100
@@ -9,7 +9,7 @@
 import sys, vtk, copy
 import pickle
 import pkg_resources
-dv3d_egg_path = pkg_resources.resource_filename(pkg_resources.Requirement.parse("dv3d"), "share/dv3d")
+dv3d_egg_path = pkg_resources.resource_filename(pkg_resources.Requirement.parse("dv3d"), "../../../share/dv3d")
 pkl_path = os.path.join( dv3d_egg_path, 'colormaps.pkl' )
 colormap_file = open( pkl_path, 'rb' )
 try:
diff -ru dv3d-8.2.1/DV3D/ConfigurationFunctions.py dv3d-8.2.1-edit/DV3D/ConfigurationFunctions.py
--- dv3d-8.2.1/DV3D/ConfigurationFunctions.py	2020-07-17 23:18:27.000000000 +0200
+++ dv3d-8.2.1-edit/DV3D/ConfigurationFunctions.py	2021-03-02 17:15:09.177821440 +0100
@@ -10,7 +10,7 @@
 import inspect, ast
 from weakref import WeakSet, WeakKeyDictionary
 import pkg_resources
-dv3d_egg_path = pkg_resources.resource_filename(pkg_resources.Requirement.parse("dv3d"), "share/dv3d")
+dv3d_egg_path = pkg_resources.resource_filename(pkg_resources.Requirement.parse("dv3d"), "../../../share/dv3d")
 
 try:
     basestring
diff -ru dv3d-8.2.1/DV3D/MapManager.py dv3d-8.2.1-edit/DV3D/MapManager.py
--- dv3d-8.2.1/DV3D/MapManager.py	2020-07-17 23:18:27.000000000 +0200
+++ dv3d-8.2.1-edit/DV3D/MapManager.py	2021-03-02 17:15:02.968846402 +0100
@@ -10,7 +10,7 @@
 import numpy as np
 import os, vtk
 import pkg_resources
-dv3d_egg_path = pkg_resources.resource_filename(pkg_resources.Requirement.parse("dv3d"), "share/dv3d")
+dv3d_egg_path = pkg_resources.resource_filename(pkg_resources.Requirement.parse("dv3d"), "../../../share/dv3d")
 
 defaultMapDir = dv3d_egg_path
 defaultMapFile = os.path.join( defaultMapDir,  'earth2k.jpg' )
diff -ru dv3d-8.2.1/DV3D/StructuredGridPlot.py dv3d-8.2.1-edit/DV3D/StructuredGridPlot.py
--- dv3d-8.2.1/DV3D/StructuredGridPlot.py	2020-07-17 23:18:27.000000000 +0200
+++ dv3d-8.2.1-edit/DV3D/StructuredGridPlot.py	2021-03-02 17:14:51.585892197 +0100
@@ -12,7 +12,7 @@
 from .DV3DPlot import *
 from .MapManager import MapManager
 import pkg_resources
-dv3d_egg_path = pkg_resources.resource_filename(pkg_resources.Requirement.parse("dv3d"), "share/dv3d")
+dv3d_egg_path = pkg_resources.resource_filename(pkg_resources.Requirement.parse("dv3d"), "../../../share/dv3d")
 
 class StructuredGridPlot(DV3DPlot):
 
